apps/web/components/sidebar/nav-chat-history.tsx

```
@@ -98,74 +99,80 @@ export function NavChatHistory() {
-                    if (e.key === "Escape") cancelRename();
-                  }}
-                  className="h-8 border-border/40 focus-visible:ring-0 focus-visible:border-border/60 shadow-none"
-                  aria-label="Edit chat title"
-                />
-              </div>
-            ) : (
-              <SidebarMenuButton asChild className="w-full justify-start">
-                <Link
-                  href={`/c/${thread.id}`}
-                  aria-label={`Open chat: ${thread.title || "New Chat"}`}
+        {true ? (
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)